easybuild/easyconfigs/v/VASP/VASP-5.4.1-intel-2016.02-GCC-4.9.eb

```diff
@@ -14,15 +14,32 @@ toolchainopts = {'usempi': True}
 # Vasp is proprietary software, see http://www.vasp.at/index.php/faqs on how to get access to the code
 sources = [SOURCELOWER_TAR_GZ]
 
+# uncommment after downloading patches from http://cms.mpi.univie.ac.at/wiki/index.php/Installing_VASP
+#patches = [
+#    'vasp.5.4.1.08072015.patch',
+#    'vasp.5.4.1.27082015.patch',
+#    'vasp.5.4.1.06112015.patch',
+#]
+
+checksums = [
+    '57c2b07d0f70987000033984e913f1a6',  # vasp.5.4.1.tar.gz
```

Comment:
Muh, I hate when upstream does that (changing the tarball without updating the version)...

Also, yes, please use the original tarball name, to avoid confusion.